### ISSUE TITLE
Octopoes worker "could not deserialize value"

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-coordination#2568]
**Describe the bug**
Lots of info logs in the form: "Could not deserialize value" in octopoes api worker. They are thrown because strings like `IPAddressV6|internet|<IPv6/>` do not contain the string "object_type" (see `ooi_repository.py`)

there is a vague comment stating:
```
# Is not an error, XTDB returns the foreign key as a string,
# when related object is not found
```

**Expected behavior**
No info log spam if there is no problem, or a fix if there is actually a problem

**OpenKAT version**
main branch 27 feb 2024 (+- 1.15)

**Comment by originalsouth** [comment]
It seem that the query yielded by `construct_neighbour_query_multi` solely yields the `xt/id` without the `object_type`...

**Comment by originalsouth** [comment]
Warning likely to be triggered by #2403  (a7e7af0c6f16a7f3fbc35c977d44131bb2524121) -- before it was likely dormant.

**Comment by originalsouth** [comment]
The query:
```
{ :query { :find [ (pull ?e [ :xt/id {:SoftwareInstance/_ooi [*]} ]) ] :in [[ _xt_id ... ]] :where [[?e :xt/id _xt_id]] } :in-args [["DNSARecord|internet|asfsdgswrgew223424.com|34.90.130.132"]] }
```
should instead be:
```
{ :query { :find [ (pull ?e [ :xt/id {:DNSARecord/address [*]} ]) ] :in [[ _xt_id ... ]] :where [[?e :xt/id _xt_id]] } :in-args [[\"DNSARecord|internet|asfsdgswrgew223424.com|34.90.130.132\"]] }
```

**Comment by underdarknl** [comment]
Since nothing seems broken, do we really want or need this data on the loctaion where now the exception is raised?

**Comment by originalsouth** [comment]
> Since nothing seems broken, do we really want or need this data on the loctaion where now the exception is raised?

It looks like `recalculate_scan_profiles` is a bit more conservative than it should be -- other than that nothing major. Still looking into the exact source of the problems so we can safely say nothing else is impacted.

**Comment by originalsouth** [comment]
Fixed in #2592.
